cvpy/annotation/cvat/CVATProject.py

```diff
@@ -70,28 +81,72 @@ def _delete_project_in_cvat(self) -> None:
         if response.status_code != HTTPStatus.NO_CONTENT:
             raise Exception(f'Unable to delete the project in the CVAT server: {response.reason}')
 
-    def post_images(self, image_table: CASTable) -> None:
+    def post_images(self, image_table: ImageTable) -> None:
         '''
         Create a CVAT task under the project and upload images from a CAS table to that task.
 
         Parameters
         ----------
         image_table: 
-            Specifies the input CAS table that contains encoded images to be uploaded.
+            Specifies the input table that contains encoded images to be uploaded.
         
         '''
-        pass
+        # Check that the image_table contains the required columns for posting images to CVAT.
+        if image_table.id == None:
+            raise Exception(f'Provided ImageTable is missing a required column: id')
+        elif image_table.image == None: 
+            raise Exception(f'Provided ImageTable is missing a required column: image')
+        elif image_table.type == None:
+            raise Exception(f'Provided ImageTable is missing a required column: type')
+
+        # Fetch the image table that will be posted to CVAT.
+        image_table_fetched = image_table.table.fetchImages(fetchImagesVars=["_id_", "_type_"])
+
+        # Create a CVATTask for this set of images.
+        task = CVATTask(str(0), str(image_table_fetched['Images'].shape[0]-1), image_table, self)
```

Comment:
Since start_image_id is always 0 and end_image_id = image_count - 1, we can just set these values in the Task class constructor. 

In the Task class, you can get the image count using: 
image_count = image_table.tableinfo().TableInfo.Rows.values[0]

We can then make start_image_id and end_image_id as optional parameters in Task constructor.